qdraw/starsky
# Feature/202402 open native

# PR Details 
## Description / Motivation and Context

-toggle first time does not update due rerender

- add UI for max amount
- add UI for selecting default app

Remote starsky:
- hide application settings if local
- Add if check in menu item
- Write in docs why - no menu item when remote
- 

Test key press feature toggle disabled 
Show notification no slection CMD e
Remote only default editing no overwrite 

Breaking change UseLocalDesktopUi -> UseLocalDesktop

Done:
- Check if collections false open xmp 
- Collections on duplicate item
- Add check max 5 backend
- Test if run as Windows / Systemd service
- - 204 - no content - error change status code > 206
- - test readonly
```
il: Microsoft.AspNetCore.Server.Kestrel[13]
      Connection id "0HN1IPHEUEDDD", Request id "0HN1IPHEUEDDD:00000001": An unhandled exception was thrown by the application.
      System.InvalidOperationException: Writing to the response body is invalid for responses with status code 204.
         at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.FirstWriteAsyncInternal(ReadOnlyMemory`1 data, CancellationToken cancellationToken)
         at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.WritePipeAsync(ReadOnlyMemory`1 data, CancellationToken cancellationToken)
         at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpResponseStream.WriteAsync(ReadOnlyMemory`1 source, CancellationToken cancellationToken)
         at System.IO.Compression.DeflateStream.WriteDeflaterOutputAsync(CancellationToken cancellationToken)
         at System.IO.Compression.DeflateStream.<WriteAsyncMemory>g__Core|66_0(ReadOnlyMemory`1 buffer, CancellationToken cancellationToken)
         at Microsoft.AspNetCore.ResponseCompression.ResponseCompressionBody.WriteAsync(ReadOnlyMemory`1 buffer, CancellationToken cancellationToken)
         at System.Text.Json.Serialization.Metadata.JsonTypeInfo`1.SerializeAsync(Stream utf8Json, T rootValue, CancellationToken cancellationToken, Object rootValueBoxed)
         at System.Text.Json.Serialization.Metadata.JsonTypeInfo`1.SerializeAsync(Stream utf8Json, T rootValue, CancellationToken cancellationToken, Object rootValueBoxed)
         at System.Text.Json.Serialization.Metadata.JsonTypeInfo`1.SerializeAsync(Stream utf8Json, T rootValue, CancellationToken cancellationToken, Object rootValueBoxed)
         at Microsoft.AspNetCore.Mvc.Infrastructure.SystemTextJsonResultExecutor.ExecuteAsync(ActionContext context, JsonResult result)
         at Microsoft.AspNetCore.Mvc.Infrastruc
```

<img width="879" alt="Schermafbeelding 2024-02-19 om 10 10 23" src="https://github.com/qdraw/starsky/assets/1826996/046d2f2f-55be-4bcd-8a0e-7bd43118bec3">

```
Do you really want to edit all of the selected photos?
```



## Related Issue






## How Has This Been Tested





- [ ] C# Unit tests 
- [ ] Typescript Unit tests 
- [ ] Other Unit tests
- [ ] Automatic End2end tests (starsky-tools/end2end)


## Types of changes



- [ ] Added _for new features_
- [ ] Breaking change _fix or feature that would cause existing functionality to change_
- [ ] Changed _for non-breaking changes in existing functionality for example docs change / refactoring / dependency upgrades_
- [ ] Deprecated _for soon-to-be removed features_
- [ ] Removed _for now removed features_
- [ ] Fixed _for any bug fixes_
- [ ] Security _in case of vulnerabilities_


## Checklist




- [ ] I have updated the documentation accordingly (update when needed)
- [ ] I have updated the **./history.md** document
